lib/Benchmark/Metadata/DocParser.php

```diff
@@ -13,13 +13,16 @@
 
 use Doctrine\Common\Annotations\AnnotationException;
 use Doctrine\Common\Annotations\DocParser as DoctrineDocParser;
+use Doctrine\Common\Annotations\PhpParser;
 
 /**
  * PHPBench doc parser, uses the Doctrine DocParser.
  */
 class DocParser
```

Comment:
Probably need to refactor here and either use the Doctrine `AnnotationReader` (there was a reason I didn't use that) or adopt the same pattern.